### PR TITLE
Added scripts for automating sending messages in Telegram

### LINKED ISSUE #323: Automate sending messages on Telegram
# Description

This script will automate the process of sending messages in Telegram

## Language

- [ ] Bash
- [ ] Go
- [ ] Javascript
- [ ] Powershell
- [x] Python
- [ ] Other (Please specify)

### PR DESCRIPTION
**Pull request by Namyalg**
# Description

This script written in Python, can be used to automate the process of sending messages in Telegram

Fixes #323 

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Automate-Telegram/automate_telegram.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#Import and dependencies
+from selenium import webdriver
+import time
+from selenium.webdriver.common.keys import Keys
+
+#Here the process of automation is achieved by using the framework Selenium.
+#Selenium is a portable framework for testing and automating web applications web applications
+
+#Path to chromedriver.exe
+chrome_path = r"C:\Users\giril\AppData\Local\Programs\Python\Python36-32\chromedriver.exe"
```
Comment (srimani-programmer) [inline Automate-Telegram/automate_telegram.py:10]:
```suggestion
# Ask for the user to enter the custom driver path. This script is not cross-platform compatible. It doesn't work in Linux and Unix based systems.
```

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
Use the proper formatting guidelines to format the code and make the appropriate changes.!

**Comment by Namyalg** [comment]
Okay, I will make the necessary changes.

**Comment by Namyalg** [comment]
I have made the necessary changes, I would like to know if I have to change anything

File: Automate-Telegram/automate_telegram.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        driver.find_element_by_xpath("/html/body/div[1]/div[2]/div/div[1]/div[1]/div/input").send_keys(name)
+        time.sleep(4)
+        driver.find_element_by_xpath("/html/body/div[1]/div[2]/div/div[1]/div[2]/div/div[1]/ul/li").click()
+        done = 1
+        while done:
+            message = input("Enter message ")
+            driver.find_element_by_xpath("/html/body/div[1]/div[2]/div/div[2]/div[3]/div/div[3]/div[2]/div/div/div/form/div[2]/div[5]").send_keys(message)
+            driver.find_element_by_xpath("/html/body/div[1]/div[2]/div/div[2]/div[3]/div/div[3]/div[2]/div/div/div/form/div[2]/div[5]").send_keys(Keys.ENTER)
+            print("Do you want to send another message? ")
+            done = int(input("Enter 1 to continue, 0 to stop "))
```
Comment (srimani-programmer) [inline Automate-Telegram/automate_telegram.py:34]:
```suggestion
            # Add the proper variable name

```

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
This script will collapse if the user has a default driver path. please add the exception case for that.!

**Comment by Namyalg** [comment]
I have made the changes

**Comment by HarshCasper** [comment]
Looks good to me now! 

I am merging this!

**Pull request merged**